yabeda-puma-plugin.gemspec

```diff
@@ -21,9 +21,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "yabeda", "~> 0.2"
+  spec.add_runtime_dependency "yabeda", "~> 0.5.0"
```

Comment:
Any specific reason why you're upgrading and constraining this version requirement?